notesapi/v1/tests/test_views.py

```diff
@@ -47,66 +53,38 @@ def setUp(self):
             ],
         }
 
-        self.expected_note = {
-            "created": "2014-11-26T00:00:00+00:00",
-            "updated": "2014-11-26T00:00:00+00:00",
-            "user": TEST_USER,
-            "usage_id": "test-usage-id",
-            "course_id": "test-course-id",
-            "text": "test note text",
-            "quote": "test note quote",
-            "ranges": [
-                {
-                    "start": "/p[1]",
-                    "end": "/p[1]",
-                    "startOffset": 0,
-                    "endOffset": 10,
-                }
-            ],
-            "permissions": {"read": ["group:__consumer__"]},
-        }
-
-    def tearDown(self):
-        Annotation.drop_all()
-
-    def _create_annotation(self, refresh=True, **kwargs):
+    def _create_annotation(self, **kwargs):
         """
-        Create annotation directly in elasticsearch.
+        Create annotation
         """
-        opts = {
-            'user': TEST_USER,
-        }
+        opts = self.payload.copy()
         opts.update(kwargs)
-        annotation = Annotation(**opts)
-        annotation.save(refresh=refresh)
-        return annotation
+        url = reverse('api:v1:annotations')
+        response = self.client.post(url, opts, format='json')
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        call_command('update_index')
+        return response.data.copy()
 
     def _get_annotation(self, annotation_id):
         """
-        Fetch annotation directly from elasticsearch.
-        """
-        return Annotation.fetch(annotation_id)
-
-    def _get_search_results(self, qs=''):
+        Fetch annotation
         """
-        Helper for search method.
-        """
-        url = reverse('api:v1:annotations_search') + '?user=' + TEST_USER + '&{}'.format(qs)
-        result = self.client.get(url)
-        return result.data
+        call_command('update_index')
```

Comment:
same question about the mgmt command here - doesn't seem like a valid test if we're constantly re-indexing everything.